web/src/main/java/no/nav/modiapersonoversikt/rest/person/PersonsokController.kt

```diff
@@ -121,7 +122,11 @@ private fun lagBostedsadresse(adr: List<SokPerson.Bostedsadresse>?): String? {
     if (adr.isNullOrEmpty()) {
         return null
     }
-    val adresse = adr.first()
+    val adresse = adr
+        .filter { it.gyldigTilOgMed?.value?.isBefore(LocalDateTime.now()) != true }
```

Comment:
Foreslår å trekke logikken her ut i en egen funksjon slik at den blir litt enklere å forstå seg på. 
```suggestion
      .filter { it.harGyldigPeriode() }
```

så kan man evt lage funksjonen som sjekker det sånn:
```kotlin
private fun SokPerson.Kontaktadresse.harGyldigPeriode(): Boolean = harGyldigPeriode(gyldigFraOgMed, gyldigTilOgMed)
private fun SokPerson.Bostedsadresse.harGyldigPeriode(): Boolean = harGyldigPeriode(gyldigFraOgMed, gyldigTilOgMed)

private fun harGyldigPeriode(start: SokPerson.DateTime?, end: SokPerson.DateTime?): Boolean {
    val startDate = start?.value ?: LocalDateTime.MIN
    val endDate = end?.value ?: LocalDateTime.MAX
    val now = LocalDateTime.now()

    return startDate.isBefore(now) && now.isBefore(endDate)
}
```